src/PostgREST/MediaType.hs

```diff
@@ -22,6 +22,7 @@ import Protolude
 -- | Enumeration of currently supported media types
 data MediaType
   = MTApplicationJSON
+  | MTApplicationNullJSON
```

Comment:
This should have another name, maybe `MTArrayJSON` similar to `MTSingularJSON`. The thing is that the `;nulls=stripped` attribute will also be added to the `MTSingularJSON` according to  https://github.com/PostgREST/postgrest/issues/1601#issuecomment-801038777, so you can add it as a field. You could check the `MTPlanAttrs` and follow it as example, it should be similar to that.

Reply:
Hi @laurenceisla i've followed your advice, i hope you can review my code.